nuxeo-java-client/src/main/java/org/nuxeo/client/api/objects/Document.java

```diff
@@ -568,4 +585,15 @@ public Task fetchTaskById(String documentId) {
         return (Task) getResponse(documentId);
     }
 
+    private void refuseIfDateFound(Object obj) {
+        if (obj instanceof Calendar || obj instanceof Date) {
+            throw new IllegalArgumentException(MSG_DATE_UNSUPPORTED);
+        }
+        if (obj instanceof Map) {
```

Comment:
I just see @efge's message. Ok, so it seems that `Collection` + array would be fine, no need to check other types.